src/views/projectExplorer/remoteIncludePath.ts

```diff
@@ -27,6 +27,9 @@ export default class RemoteIncludePath extends IFSDirectory {
     );
 
     this.contextValue = RemoteIncludePath.contextValue;
+    if (!custom || !custom.description) {
+      this.contextValue += ContextValue.configurable;
+    }
```

Comment:
what does `this.contextValue += ContextValue.configurable;` do?

Reply:
`ContextValue.configurable` is used to control which libraries or include path tree items have the `Configure as Variable` action. If the value is hardcoded, it will have this context value.